MQTTClient/MQTTClient/MQTTCFSocketTransport.m

```diff
@@ -40,6 +42,23 @@ - (void)dealloc {
     [self close];
 }
 
+- (void)setQueue:(dispatch_queue_t)queue {
```

Comment:
Is this called even if queue is set in initialiser?

Reply:
Yes it is